bindings/java/pom.xml

```diff
@@ -183,7 +183,10 @@
         <extensions>true</extensions>
         <configuration>
           <javahOutputDirectory>${basedir}/src/main/native/</javahOutputDirectory>
-          <javahClassNames>org.ucx.jucx.ucp.UcpContext</javahClassNames>
+          <javahClassNames>
+            <javahClassName>org.ucx.jucx.UcxConstants</javahClassName>
```

Comment:
maybe it should be "UcpConstants"?

Reply:
I thought to set in future other constants. E.g. Ucs (Tthread mode, etc.).

Reply:
maybe each should have own class?

Reply:
Ok, will do. Took as example MPI, where all constants in the same file: https://github.com/open-mpi/ompi/blob/master/ompi/mpi/java/java/Constant.java

Reply:
For MPI, it makes more sense, since it's since it's same "api layer"